pkg/sql/sqlbase/structured.go

```diff
@@ -3239,6 +3380,12 @@ func (desc *MutableTableDescriptor) AddPrimaryKeySwapMutation(swap *PrimaryKeySw
 	desc.addMutation(m)
 }
 
+// AddComputedColumnSwapMutation adds a ComputedColumnSwap mutation to the table descriptor.
+func (desc *MutableTableDescriptor) AddComputedColumnSwapMutation(swap *ComputedColumnSwap) {
```

Comment:
I think theres a missing case here for reverseMutations, which is called when a schema change job runs into an error or is cancelled.